cmd/proxy/actions/index.go

```diff
@@ -0,0 +1,48 @@
+package actions
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/index"
+)
+
+// indexHandler implements GET baseURL/index
+func indexHandler(index index.Indexer) http.HandlerFunc {
+	const op errors.Op = "actions.IndexHandler"
+	return func(w http.ResponseWriter, r *http.Request) {
+		var (
+			err   error
+			limit int
+			since time.Time
+		)
+		if limitStr := r.FormValue("limit"); limitStr != "" {
+			limit, err = strconv.Atoi(limitStr)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+		}
+		if sinceStr := r.FormValue("since"); sinceStr != "" {
+			since, err = time.Parse(time.RFC3339, sinceStr)
```

Comment:
Looks like you set this error, but never check it

Reply:
Nice catch, will definitely write handler tests after this PR 😄